models/pathology_nuclei_segmentation_classification/docs/README.md

```diff
@@ -1,13 +1,11 @@
 # Model Overview
 A pre-trained model for simultaneous segmentation and classification of nuclei within multi-tissue histology images based on CoNSeP data. The details of the model can be found in [1].
 
-The model is trained to simultaneously segment and classify nuclei. Training is done via a two-stage approach. First initialized the model with pre-trained weights on the [ImageNet dataset](https://ieeexplore.ieee.org/document/5206848), trained only the decoders for the first 50 epochs, and then fine-tuned all layers for another 50 epochs. There are two training modes in total. If "original" mode is specified, [270, 270] and [80, 80] are used for `patch_size` and `out_size` respectively. If "fast" mode is specified, [256, 256] and [164, 164] are used for `patch_size` and `out_size` respectively. The results shown below are based on the "fast" mode.
+The model is trained to simultaneously segment and classify nuclei. Training is done via a two-stage approach. First initialized the model with pre-trained weights, trained only the decoders for the first 50 epochs, and then fine-tuned all layers for another 50 epochs. There are two training modes in total. If "original" mode is specified, [270, 270] and [80, 80] are used for `patch_size` and `out_size` respectively. If "fast" mode is specified, [256, 256] and [164, 164] are used for `patch_size` and `out_size` respectively. The results shown below are based on the "fast" mode.
```

Comment:
The model is trained to simultaneously segment and classify nuclei, and a two-stage approach is utilized:

- initialize the model with pre-trained weight, and train the decoder only for 50 epochs.
- finetune all layers for another 50 epochs.

There are two training modes in total ...

Reply:
Updated it in this [commit](https://github.com/Project-MONAI/model-zoo/pull/370/commits/70c334bd898c5e11c45522f5123fb0d4c60bc5f8), thanks!